arch/platform/native/platform.c

```diff
@@ -238,7 +247,18 @@ platform_init_stage_one()
   gpio_hal_init();
   button_hal_init();
   leds_init();
-  return;
+#if defined(__linux__) || defined(__MACH__)
+  struct csprng_seed seed;
+#ifdef __linux__
+  if(getrandom(seed.u8, CSPRNG_SEED_LEN, GRND_RANDOM) != CSPRNG_SEED_LEN) {
```

Comment:
The Windows support has been removed, and FreeBSD has getrandom, so I think the inner ifdef can be `#ifdef __APPLE__ ... #else ... #endif`, and the outer ifdef can be dropped.